### DISCUSSION TITLE
Wipe out old wiki

### DISCUSSION THREAD

**Discussion by raffaelemancuso** [hyprwm/Hyprland#1029] [General]
Can you wipe out the inner pages of https://github.com/hyprwm/Hyprland/wiki, leaving behind only the home page?

The inner pages show up in google search result without any warning that they are no longer valid, and it's very confusing.

**Comment by vaxerski** [comment]
done bby

**Comment by PeterPan94** [comment]
This is still an issue. https://wiki.hypr.land/Configuring/ serves an old version of the wiki and is linked to by search engines (at least by duckduckgo).

I think that forwarding to the wiki homepage or at least marking the pages as stale (as not to serve 404s to clients coming from search engines) may be a solution.

**Reply by erstarr** [reply]
While after the wiki rewrite the old wiki IS still there and this IS an issue, necro-ing a post from 2022 about an entirely different topic is certifiably insane.

**Reply by PeterPan94** [reply]
I spend an huge amount of my free time trying to find out, how I was even supposed to report this issue. After the tenth reminder to look at open and closed discussions for duplicates I finally gave in and posted here trying to help.

Maybe make it clearer how you want me to report this. And being a bit nicer would aid the cause tremendously to make me consider to ever helping you again.

Also, maybe I misread the issue a little bit as it sounds very close to my problem. It seems _certifiable insane_ to believe that this kind of community interaction is anything other than destructive.

**Reply by wabarTM** [reply]
> I spend an huge amount of my free time trying to find out, how I was even supposed to report this issue.

Open a discussion

> After the tenth reminder to look at open and closed discussions for duplicates I finally gave in and posted here trying to help.

It is 2 years old

> Maybe make it clearer how you want me to report this.

Open a fucking discussion

> to make me consider to ever helping you again.

You did not helped anyone in any way

> It seems certifiable insane to believe that this kind of community interaction is anything other than destructive.

No

**Reply by PeterPan94** [reply]
> Open a discussion
> Open a fucking discussion
Warnings of not duplicating are everywhere. You're supposed to search through everything first. Sorry for doing this.
> It is 2 years old
I missed where in the guide this was mentioned as relevant
> You did not helped anyone in any way
This is very clear. You should maybe consider that I at least was trying to.

You seem to be desensitized by AI reports. AI doesn't mind rudeness. I do.

**Reply by erstarr** [reply]
You are not owed a paragraph of platitudes and niceties every time someone engages with you.

This Guthub Discussion is:

From 2022 - 4 whole years old

From the time when wiki was in github -- as you might have noticed had you read the original text and clicked on the link there.
(In fact i was not here when this discussion was opened? Wanna know how i figured this out? By reading the original discussion body.)

Doesn't have anything to do with the current scenario other than the keywords "wiki" and "old". You are human, it is natural to expect your thinking process to be deeper than keyword matching.

To put this in even simpler terms: this is a post about wiping the wiki in the GitHub repo after it is moved to its own website.
Which has nothing to do with the current scenario of the wiki website having a cached version of the wiki from before we rewrote it still being accessible to users OTHER than the 2 keywords i mentioned above.

It takes a trivial amount of thinking to connect the dots and come to the trivial conclusion that commenting about this here, on a Discussion that's 4 YEARS OLD to boot, is illogical.

Warnings of duplicates are indeed everywhere, and just as well because you are not meant to dupe - this doesn't mean, however, that you should comment on issues/discussions that are not dupes of your subject matter.

>  I missed where in the guide this was mentioned as relevant
 
 Let me elucidate you to the wonders of the English language.
 
 "[This is still an issue](https://github.com/hyprwm/Hyprland/discussions/1029#discussioncomment-18466804:~:text=This%20is%20still%20an%20issue)"
 
Means this WAS and still IS an issue. `This` refers to the subject matter of this here GitHub Discussion. This statement is objectively wrong.
 
 Thank you for attending my TED Talk.
 
> This is very clear. You should maybe consider that I at least was trying to.

We remain thankful to those that help, or at least try to. But while intentions are important, good intentions don't render you immune to criticism. You failed to make the most basic logical leaps; good intentions or not; you shouldn't take offence when someone tells you what you've done here is hardly worth praise. This is not mean, or rude, or an attack on your character or honour; it's just fact. Cold, hard, emotionless fact.

Not every single word that is not praise, and is not preceded by a whole novella of assurances of no offence, assurances of how positive of a change you are making by even trying to help, and every other overly overdone attempts to reinforce inoffensiveness and positivity is meant to insult you, offend you, or demean you. Take it at face value and move on.

**Reply by PeterPan94** [reply]
> You are not owed a paragraph of platitudes and niceties every time someone engages with you.

To retrain from slander was all I asked for. (You called me "clinically insane" in the first response – and it didn't stop there. Leaving these words out would even save time.)

> From the time when wiki was in github -- as you might have noticed had you read the original text and clicked on the link there.

I actually looked at the original post and thought that "wiping out inner pages" was supposed to be done by the code syncing the github repo to the wiki before copying the new pages. Meaning it was describing the same issue I was having, but when the wiki was still hosted on github. When understood this way, my posting here becomes at least understandable – even if shortsighted.

I appreciate your detailed response. I opened a new discussion about this topic: https://github.com/hyprwm/Hyprland/discussions/16279
